src/pages/users/changePassword.tsx

```diff
@@ -0,0 +1,44 @@
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { fas } from "@fortawesome/free-solid-svg-icons";
+import React, { FormEvent, useState } from "react";
+import InputText from "@/elements/inputText/inputText";
+import styles from "./signForm.module.scss";
+import { formFieldByName } from "@/data/formFields";
+import FormHeader from "./formHeader";
+
+library.add(fas);
+
+interface IChangePasswordProps {
+  onExit: (() => void) | undefined;
+}
+
+const useFields: () => [
+  string,
+  React.Dispatch<React.SetStateAction<string>>,
+  string,
+  React.Dispatch<React.SetStateAction<string>>
+] = () => {
+  const [password, setPassword] = useState("");
+  const [password2, setPassword2] = useState("");
+
+  return [password, setPassword, password2, setPassword2];
+};
+
```

Comment:
the same with custom hook. in this case you just duplicate your code

Reply:
also rewritten